bindings/python/iota_sdk/utils.py

```diff
@@ -31,11 +32,11 @@ def bech32_to_hex(bech32: str) -> HexStr:
         })
 
     @staticmethod
-    def hex_to_bech32(hex: HexStr, bech32_hrp: str) -> str:
+    def hex_to_bech32(hex_str: HexStr, bech32_hrp: str) -> str:
```

Comment:
not a big fan of calling variable names after their type tbh :shrug: I would rather give it a meaningful name that can stay the same even if the type changes.